### PR TITLE
Feature/attachments

### PR DESCRIPTION
**Pull request by vigan-abd**
- Added checks for attachments in sendEmail api action
- Updated test packages and fixed linting issues
- Updated docs and added example file

### CODE REVIEW

File: test/integration.js [hunk 1]
```diff
@@ -58,11 +59,19 @@ describe('Sendgrid test', () => {
       from: 'test@example.com',
       subject: 'Sending with SendGrid is Fun',
       text: 'and easy to do anywhere, even with Node.js',
-      html: '<strong>and easy to do anywhere, even with Node.js</strong>'
+      html: '<strong>and easy to do anywhere, even with Node.js</strong>',
+      attachments: [
+        {
+          content: fs.readFileSync(path.join(__dirname, 'data/bitfinex.png')).toString('base64'),
+          filename: 'bitinex.png',
```
Comment (bariscicek) [inline test/integration.js:66]: Typo, not important but still.

**Review by bariscicek — APPROVED** [review]
There is a minor typo but not a blocker.

File: test/integration.js (hunk 1 above)
Comment (vigan-abd) [reply test/integration.js]: fixed :)

**Review by bariscicek — APPROVED** [review]

**Review by robertkowalski — APPROVED** [review]
lgtm, i guess attachment size is restricted by cloudflare?

**Pull request merged**